src/components/basic/text/Text.stories.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { Meta, StoryObj } from '@storybook/react';
+
+import { Text } from './Text';
+import { colorArgTypeControl, themeFontSize, themeFontWeight } from '../../../docs/utils';
+
+const meta = {
+	component: Text
+} satisfies Meta<typeof Text>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const Common: Partial<Story> = {
+	argTypes: {
+		color: colorArgTypeControl,
+		weight: {
+			control: 'radio',
+			options: themeFontWeight
+		},
+		size: {
+			control: 'radio',
+			options: themeFontSize
+		}
+	},
+	args: {
+		children: 'Lorem Ipsum dolor sit amet'
+	}
+};
+
+export const Size: Story = {
```

Comment:
Create a "base"/"default" story as first, with only the required props set, because the first one is the one used for the documentation example, and imo having a "clean/default example" is more useful there